From 3267334937464b9c84b812f20fb12cab8b8dd0d5 Mon Sep 17 00:00:00 2001
From: Mirza Krak <mirza.krak@gmail.com>
Date: Sun, 30 Oct 2016 21:39:10 +0100
Subject: [PATCH 1/1] CONFIGS: rpi: enable mender.io requirements

Which are CONFIG_BOOTCOUNT_ENV and CONFIG_BOOTCOUNT_LIMIT.

Mender.io also requires that FAT_ENV_INTERFACE, FAT_ENV_DEVICE_AND_PART
and FAT_ENV_FILE are not present in the configuration.
---
 include/configs/rpi-common.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/include/configs/rpi-common.h b/include/configs/rpi-common.h
index 2c3b026..505a2fe 100644
--- a/include/configs/rpi-common.h
+++ b/include/configs/rpi-common.h
@@ -107,16 +107,13 @@
 /* Environment */
 #define CONFIG_ENV_SIZE			SZ_16K
-#define CONFIG_ENV_IS_IN_FAT
-#define FAT_ENV_INTERFACE		"mmc"
-#define FAT_ENV_DEVICE_AND_PART		"0:1"
-#define FAT_ENV_FILE			"uboot.env"
-#define CONFIG_FAT_WRITE
+#define CONFIG_ENV_IS_IN_MMC
 #define CONFIG_ENV_VARS_UBOOT_CONFIG
 #define CONFIG_SYS_LOAD_ADDR		0x1000000
 #define CONFIG_CONSOLE_MUX
 #define CONFIG_SYS_CONSOLE_IS_IN_ENV
 #define CONFIG_PREBOOT			"usb start"
+#define CONFIG_BOOTCOUNT_ENV
+#define CONFIG_BOOTCOUNT_LIMIT
 
 /* Shell */
 #define CONFIG_SYS_MAXARGS		16
-- 
2.1.4

